docs-kits/kits/ESS-Kit/page_adoption-view.md

```diff
@@ -111,23 +119,23 @@ The trace of an incident follows previously mentioned principles along any oblig
 | Operational internal role/ central function that covers incidents as primary responsibility | Receives incident from various sources |
 | First contact and dispatcher for ESS-related incidents and incident status | Reduce fraud / identify fake incidents |
 | | Consolidate incidents (no multiples) |
-| | Set / maintain central incident status and anonymized information (respect: one-up/one-down, decentral, data-sovereign, "flag" to allow sharing) |
+| | Set / maintain central incident status and anonymized information (resp. one-up/one-down, decentral, data-sovereign, "flag" to allow sharing) |
```

Comment:
resp. could be misread as responsible or others - unless there is a technical reason for the shortening, please leave as is.

Reply:
changed to respectively